runFACTS.py

```diff
@@ -131,6 +135,47 @@ def print_experimentsteps(experimentsteps):
         print_pipeline(pipelines)
         print('')
 
+def print_experimentsteps_script(experimentsteps):
+
+    print('#!/bin/bash\n')
+
+    print('if [ -z "$WORKDIR" ]; then  ')
+    print('   WORKDIR=/scratch/`whoami`/test.`date +%s`')
```

Comment:
It would be useful to include the name of the module. I think that could be done by extracting `pipeline_id` and add it the working directory name.